proto/cometbft/abci/v2/types.proto

```
@@ -1,13 +1,14 @@
 syntax = "proto3";
-package cometbft.abci.v1;
+package cometbft.abci.v2;
 
 import "cometbft/crypto/v1/proof.proto";
```

Buf (1.54.0)

import "cometbft/crypto/v1/proof.proto": file does not exist

(COMPILE)